pom.xml

```diff
@@ -15,29 +14,27 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <groupId>org.codehaus.plexus</groupId>
     <artifactId>plexus</artifactId>
-    <version>10</version>
+    <version>12-SNAPSHOT</version>
   </parent>
 
   <artifactId>plexus-utils</artifactId>
   <version>4.0.0-SNAPSHOT</version>
 
   <name>Plexus Common Utilities</name>
   <description>A collection of various utility classes to ease working with strings, files, command lines and
-    more.
-  </description>
+    more.</description>
 
   <scm>
     <connection>scm:git:git@github.com:codehaus-plexus/plexus-utils.git</connection>
     <developerConnection>scm:git:git@github.com:codehaus-plexus/plexus-utils.git</developerConnection>
```

Comment:
we need the same protocol like in parent - https in other case links in generated site will be broken